### DISCUSSION TITLE
Convert Audible m4p to mp3

### DISCUSSION THREAD

**Discussion by Nathalie52** [openaudible/openaudible#1338] [General]
I made a mistake of converting aax to m4b on OpenAudible, and ended up not being able to listen to the books on my iPhone. Several articles on the internet said just use iTunes, but iTunes has disappeared years ago on iOS devices. Is there a way to convert the now m4b books to mp3? Thank you.

**Comment by openaudible** [comment]
If the file is a valid M4B or M4A file it should be able to be converted. I received your sample file and it was corrupted and not readable by FFMPEG or VLC, the most common audio processing programs.

**Comment by Nathalie52** [comment]
Oh no! Thank you for your explanation. I wonder what happened that
corrupted it and other, but the case is closed so that’s that.

Thank you again for your time.

**Comment by openaudible** [comment]
That M4B wasn't created with OpenAudible. Maybe it was downloaded from the internet or grabbed from another source. 
Hopefully you can delete it and find it in your Audible library. 
Marking as resolved.

**Comment by Nathalie52** [comment]
No, it’s an Audible book but sold in Apple Books app. The first 12 books of
Shawn Inmon’s Middle Falls series, sold at $1.99 each. I don’t think Apple
sold things illegally. I paid through iCloud account. The days before,
Audible Plus withdrew them from the list of free listening. Two weeks after
that, Audible Plus released new version of the same books in bundle, not
individual anymore.

That’s why when you said it’s corrupted, I thought the timing matches
Audible’s decision to withdraw. Perhaps it’s coincidence, but why giving
Apple the right to sell them?

**Comment by Nathalie52** [comment]
I’m looking at iCloud purchases receipts right now. They’re $0.99 each, not
$1.99.

**Comment by Nathalie52** [comment]
Attached are 2 out of 12 receipts:

**Comment by openaudible** [comment]
Apple Books are not supported. They are encrypted and can’t be played on anything but Apple hardware/software. We only support Audible.

**Comment by Nathalie52** [comment]
But the books are Audible books as indicated on the cover “Only from
Audible”. They were in Audible Plus before being sold for a very limited
time (2 weeks IIRC) in Apple. They’re again being sold in Audible in bundle
of 3 as of right now.
